config/routes.rb

```diff
@@ -117,14 +116,14 @@ def patch(*)
           put "primary" => "groups#set_primary"
         end
       end
-      resources :groups, except: [:create], constraints: AdminConstraint.new do
+      resources :groups, only: [:destroy], constraints: AdminConstraint.new do
```

Comment:
Is there a route for `groups#create`?

Reply:
For some reason group methods are split between the groups controller and the admin groups controller.

Reply:
Yes there is a route for groups#create still. It's in the admin groups controller:

<img width="1021" alt="image" src="https://github.com/discourse/discourse/assets/1490496/69a5dab4-3f51-4d1a-8c5c-e3187bc0a25a">

Reply:
also this is my diff of the unused groups route changes

```
diff --git a/routes.txt b/routes.txt
index 0b4e710..55e7c50 100644
--- a/routes.txt
+++ b/routes.txt
@@ -55,12 +55,7 @@
                         primary_admin_group PUT      /admin/groups/:id/primary(.:format)                                         admin/groups#set_primary {:format=>/(json|html|\*\/\*)/}
                                admin_groups POST     /admin/groups(.:format)                                                     admin/groups#create {:format=>/(json|html|\*\/\*)/}
     automatic_membership_count_admin_groups PUT      /admin/groups/automatic_membership_count(.:format)                          admin/groups#automatic_membership_count {:format=>/(json|html|\*\/\*)/}
-                                            GET      /admin/groups(.:format)                                                     admin/groups#index {:format=>/(json|html|\*\/\*)/}
-                            new_admin_group GET      /admin/groups/new(.:format)                                                 admin/groups#new {:format=>/(json|html|\*\/\*)/}
-                           edit_admin_group GET      /admin/groups/:id/edit(.:format)                                            admin/groups#edit {:format=>/(json|html|\*\/\*)/}
-                                admin_group GET      /admin/groups/:id(.:format)                                                 admin/groups#show {:format=>/(json|html|\*\/\*)/}
-                                            PUT      /admin/groups/:id(.:format)                                                 admin/groups#update {:format=>/(json|html|\*\/\*)/}
-                                            DELETE   /admin/groups/:id(.:format)                                                 admin/groups#destroy {:format=>/(json|html|\*\/\*)/}
+                                admin_group DELETE   /admin/groups/:id(.:format)                                                 admin/groups#destroy {:format=>/(json|html|\*\/\*)/}
                                             GET      /admin/groups/:type(.:format)                                               admin/groups#show {:format=>/(json|html|\*\/\*)/}
                                             GET      /admin/groups/:type/:id(.:format)                                           admin/groups#show {:format=>/(json|html|\*\/\*)/}
                            list_admin_users GET      /admin/users/list(.:format)                                                 admin/users#index {:format=>/(json|html|\*\/\*)/}
                            ```